sdk/mx.sdk/mx_sdk_vm_impl.py

```diff
@@ -1898,7 +1906,7 @@ def _get_launcher_args():
             return ''
 
         def _get_add_exports():
-            res = self.subject.native_image_config.get_add_exports(_known_missing_jars)
+            res = ' '.join(self.subject.native_image_config.get_add_exports(_known_missing_jars))
```

Comment:
off-topic:
I am a little confused by the concept of known_missing_jars. Based on the comment above their definition it seems they are optional runtime dependencies. Shouldn't the variable then be called something like optional_runtime_depedencies, optional_runtime_deps or optional_svm_runtime_deps ? When I read known_missing_jars, it's not clear to me what the purpose is and it evokes the thought that there is probably some sort of implementation defect involved and this variable is a patch over it.

Reply:
The whole "concept" of `_known_missing_jars` seems very ad-hoc. I'm just using it here as it makes sense to get rid of `--add-exports` that would be hurtful if the respective jars are in fact missing. If there would be an optional jar that would actually need `--add-export`s we would be in trouble.

> it's not clear to me what the purpose is and it evokes the thought that there is probably some sort of implementation defect involved and this variable is a patch over it

I agree. but cleaning this up (introducing proper handling of optional jars) is beyond the scope of this PR. It would be an entire PR on its own ... seems someone is volunteering to address this :wink: